pentaho-database-gwt/src/org/pentaho/ui/database/event/DataHandler.java

```diff
@@ -1567,11 +1569,10 @@ private void setConnectionSpecificInfo( IDatabaseConnection meta ) {
     }
 
     if ( webAppName != null ) {
-
-      if ( databaseConnection != null && databaseConnection.getDatabaseName() != null && !databaseConnection.getDatabaseName().isEmpty() ) {
-        webAppName.setValue( databaseConnection.getDatabaseName() );
+      if ( StringUtils.isEmpty( databaseConnection.getName() ) ) {
+        webAppName.setValue( DEFAULT_WEB_APPLICATION_NAME );
```

Comment:
Your check above switched from .getDatabaseName to .getName-- I'm assuming the name of the connection.  Was that intentional?  Also, is it safe to remove the null checks on databaseConnection /  getDatabaseName?

The desired behavior with 8360 is that it's possible to have an empty webappname-- is that met with this change?